docs/types/mapping-types.rst

```diff
@@ -4,7 +4,8 @@
 Mapping Types
 =============
 
-You declare mapping types with the syntax ``mapping(_KeyType => _ValueType)``.
+Mapping types use the syntax ``mapping(_KeyType => _ValueType)`` and variables
+are declared as a mapping type using the syntax ``mapping (_KeyType => _ValueType) _VariableModifiers _VariableName``.
```

Comment:
It's probably not too useful to keep this.

Reply:
I still do not think it is useful to keep this.

Reply:
Which bit? It was here already, and from the conversations I've had around, people are really confused by the mapping syntax, so I'd love to try and be as explicit and clear as possible. Any suggestions to keep that clarity but express it differently?

Reply:
I think it would be better to give an example how to declare a specific variable instead of just adding random placeholders whose meaning is not explained.